### PR TITLE
Aligning render props type definition with implementation.

### PR DESCRIPTION
**Pull request by TimPerry**
Props are always provided for custom render functions.

Login: https://github.com/anthonyjgrove/react-google-login/blob/master/src/google-login.js#L128
Logout: https://github.com/anthonyjgrove/react-google-login/blob/master/src/google-logout.js#L88

### CODE REVIEW

**Comment by TimPerry** [comment]
Also noticed that disabled on login render was missing, so added that as well.

File: index.d.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export interface GoogleLogoutProps {
   readonly disabled?: boolean;
   readonly disabledStyle?: CSSProperties;
   readonly tag?: string;
-  readonly render?: (props?: { onClick: () => void }) => JSX.Element;
+  readonly render?: (props: { onClick: () => void }) => JSX.Element;
```
Comment (anthonyjgrove) [inline index.d.ts]: Can you add the disabled prop to Logout as well?

**Comment by anthonyjgrove** [comment]
Thanks!

**Comment by TimPerry** [comment]
Cheers :)

**Pull request merged**